lib/src/country_coder_super.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:point_in_polygon/point_in_polygon.dart';
+import 'package:latlong2/latlong.dart';
+
+abstract class CountryCoderSuper{
+
+  List<Map<String, dynamic>>  get countries;
```

Comment:
Should contain a list of `GeoFeature` as I defined later or perhaps rename it to `GeoRegion`, GeoArea` or `GeoBoundary`. `resolveCountry` should return this class as well.